packages/cli/package.json

```diff
@@ -28,6 +28,7 @@
     "degit": "^2.8.4",
     "dotenv": "^16.4.5",
     "fs-extra": "^10.1.0",
+    "ora": "5.4.1",
```

Comment:
question: Is it necessary? I ask because it may end up like the `chalk` package that we removed recently

Reply:
The `chalk` package was removed from **core** package. CLI dependencies should not influence the size of the core bundle. I can check if the **oclif** library has some utility for ui/ux feedbacks.

Reply:
@lariciamota . We have a problem that we are using **oclif v1** . There's even a solution for animations: 
https://github.com/oclif/cli-ux?tab=readme-ov-file#cliaction . But see that it is an archived repository and I don't know if maintaining code that uses an archived module would be a good practice.

Reply:
We could have an initiative to update oclif one day.